tests/simple-mpp/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("com.apollographql.apollo3")
+  id("org.jetbrains.kotlin.multiplatform")
+}
+
+configureMppTestsDefaults()
+
+kotlin {
+  sourceSets {
+    val commonMain by getting {
+      dependencies {
+        implementation("com.apollographql.apollo3:apollo-runtime")
+        implementation("com.apollographql.apollo3:apollo-normalized-cache")
+      }
+    }
+
+    val commonTest by getting {
+      dependencies {
+        implementation("com.apollographql.apollo3:apollo-mockserver")
+        implementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4")
+      }
+    }
+  }
+}
+
+apollo {
+  packageName.set("com.example")
+}
```

Comment:
That's a new simple mpp integration test. I used it to do some quick tests. More generally, it's handy to reproduce issues or just as a copy/paste template for other tests